mayavi/tools/pipe_base.py

```diff
@@ -167,8 +167,8 @@ def __init__(self, parent, **kwargs):
     def set(self, trait_change_notify=True, **traits):
```

Comment:
This can probably be renamed to `trait_set()` now. The intention was for it to override `HasTraits.set()`, which was renamed to `trait_set()`, so this override should follow suit.

Reply:
For the sake of backwards compatibility, there shouldn't be any harm in retaining "set" just for the `MlabSource`?